cherryontech/unmatched
# #UM-17 Refactoring profiles

## Description

This PR adds higher quality images and modifies `Card`/`SingleCard` components to match the design more.

## Related Tickets & Documents
UM-17 

## Acceptance Criteria



## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
|     | :sparkles: New feature     |
|✓| 🎨 Style                   |
|✓| :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|     | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Mobile & Desktop Screenshots/Recordings
![Desktop screenshot of unmatched project professionals page](https://github.com/cherryontech/unmatched/assets/106587342/994b0a69-ae20-4187-8298-79d042ca417f)



## What I learned


## Testing steps
1. Install dependencies: `npm i`
2. Run `git checkout feature/UM-14-bio-page`
3. Run localhost: `npm run dev`
4. View the app in browser and navigate to `/professionals` page. View the page on full screen and reduced screen width. Hover on each card to see the enlarged hover effect.


## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
|     | 🙅 no documentation needed    |


## What gif best describes this PR or how it makes you feel?
  ![Shaun The Sheep Movie Ok GIF](https://media.giphy.com/media/tIeCLkB8geYtW/giphy.gif)
